<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="mainScreen">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>640</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>826</width>
         <height>656</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Load</string>
       </attribute>
       <widget class="QPushButton" name="RenderLoadButton_Suf">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>530</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>LoadSuf</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SaveImgButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>ScreenShot</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ExportViewButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>ExportView</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ImportViewButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>ImportView</string>
        </property>
       </widget>
       <widget class="QPushButton" name="RenderSmoothTransitionButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>70</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>ST</string>
        </property>
       </widget>
       <widget class="QSlider" name="SmoothTransitionSlider">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="StopRecordButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop Record</string>
        </property>
       </widget>
       <widget class="QPushButton" name="StartRecordButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Record</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PlayVideoButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>PlayVideo</string>
        </property>
       </widget>
       <widget class="QPushButton" name="TimerControlStartButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>StartControl</string>
        </property>
       </widget>
       <widget class="QPushButton" name="LoadViewSequenceButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>LoadViewSeq</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ViewModeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>310</y>
          <width>104</width>
          <height>26</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Smoothed Cells</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Global View</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CrossSection View</string>
         </property>
        </item>
       </widget>
       <widget class="QSpinBox" name="MMPIckSurfspinBox">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>313</y>
          <width>47</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
       <widget class="QComboBox" name="MMCellcomboBox">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>280</y>
          <width>104</width>
          <height>26</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="MMTopocomboBox">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>280</y>
          <width>91</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QSlider" name="MMPickSurfSlider">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>340</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="MMNonMCurveBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>390</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>NonMCurve</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="MMCrossSectionBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>370</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CrossSection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="MMShowAllCellBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>410</y>
          <width>89</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>AllCell</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="RenderLoadButton_Topo">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>570</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>LoadTopo</string>
        </property>
       </widget>
       <widget class="QPushButton" name="RenderLoadButton_Contour">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>490</y>
          <width>115</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>LoadContour</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="FlipMaterialcheckBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>440</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>FlipMaterial</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>340</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Render</string>
       </attribute>
       <widget class="QSlider" name="disksizeSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>300</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="UnitNormalBox">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>UnitNormal</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="RenderSurfaceBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>89</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Surface</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSlider" name="RenderNLengthSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>200</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>340</y>
          <width>59</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ambient</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>260</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Tube</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>380</y>
          <width>59</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Specular</string>
        </property>
       </widget>
       <widget class="QSlider" name="ballsizeSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>280</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>17</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="RenderNormalBox">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>110</y>
          <width>89</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Normal</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>360</y>
          <width>59</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Diffuse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="RenderFlipNormal">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>150</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>FlipNormal</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QSlider" name="RenderDecadeSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>400</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>200</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Length</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>400</y>
          <width>59</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Decade</string>
        </property>
       </widget>
       <widget class="QSlider" name="RenderSpecularSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>380</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>220</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="RenderWireBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>89</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Wire</string>
        </property>
       </widget>
       <widget class="QSlider" name="tubesizeSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>260</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="RenderNLengthMultiplyer">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>180</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Multiply</string>
        </property>
       </widget>
       <widget class="QSlider" name="RenderUpNormalSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>240</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="RenderDoubleSide">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>DoubleSide</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="RenderMarkerBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>89</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Markers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSlider" name="RenderAmbientSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>340</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>280</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ball</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>UpNormal</string>
        </property>
       </widget>
       <widget class="QSlider" name="RenderLWidthSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>220</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="RenderDiffuseSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>360</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>75</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Disk</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA">
    <property name="title">
     <string>a</string>
    </property>
   </widget>
   <addaction name="menuA"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
